packages/wrangler/src/deploy/deploy.ts

```diff
@@ -737,6 +738,8 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 					})
 				);
 
+				logger.log("Script Startup Time: ", result.startup_time_ms, "ms");
```

Comment:
We should say "Worker" rather than "Script"

Reply:
Switched it